docs/tutorial_tests.md

```diff
@@ -26,22 +27,26 @@ def get_dependencies(self):
 
 We'll explore more of what DependencyHelper can do in later demos. Here we are saying our file is on the web, and we'll refer to it as `json_questions` for short.
 
-The `make_test_items` method is where we convert our `an_example.jsonl` file into TestItems. `DependencyHelper` manages downloading the file and provides us with a path to it:
+The **`make_test_items`** method is where we convert our `an_example.jsonl` file into TestItems. `DependencyHelper` manages downloading the file and provides us with a path to it:
 
 ```py
 with open(dependency_helper.get_local_path("jsonl_questions"), "r") as f:
 ```
 
 We want each `question` to go to the SUT, so we construct `Prompt(text=data["question"])`. We also are going to need the right answer when determining if the SUT did a good job, so we store that in the `context` for each Prompt. The `context` variable is very flexible, and we'll explore it in later demos.
 
-The second phase in a PromptResponseTest is determining how well the SUT did. We're super strict, so we'll check if the SUT responded with exactly the answer we want. In `measure_quality`, we get back each TestItem, but now with data about what the SUT did:
+### 2. Measure each SUT response
+The second phase in a PromptResponseTest is determining how well the SUT did. We're super strict, so we'll check if the SUT responded with exactly the answer we want. In **`measure_quality`**, we get back each TestItem, but now with data about what the SUT did:
 
 ```py
 interaction.response.completions[0].text == interaction.prompt.context
 ```
 
-We can then use one of the provided [aggregation functions](../newhelm/aggregations.py) to determine how often the SUT responded correctly.
+### 3. Aggregate the measurements
 
+We can then use one of the provided [aggregation functions](../newhelm/aggregations.py) to determine how often the SUT responded correctly in **`aggregate_measurements`**.
```

Comment:
While you are here, can you call out that the user could write their own if none of the existing ones worked?

Reply:
Does my edit convey what you had in mind? Or did you mean the user can write their own function outside of aggregate_measurements, like in aggregations.py or something?